Realm.xcodeproj/xcshareddata/xcschemes/TestHost.xcscheme

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That Twitter thread implies the issue is fixed in Xcode 7.1.1+. Is it? If not, filing a Radar would be a good idea.